lib/multi-write.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 balena.io
+ * Copyright 2019 balena.io
```

Comment:
Why though? If the file's been *created* in 2018, it's fine

Reply:
I updated the files I modified, switched back to the previous value.